contrib/completion/bash/docker

```diff
@@ -5486,11 +5486,8 @@ _docker_wait() {
 	_docker_container_wait
 }
 
-COMPOSE_PLUGIN_PATH=$(docker info --format '{{json .ClientInfo.Plugins}}' | sed -n 's/.*"Path":"\([^"]\+docker-compose\)".*/\1/p')
-
-_docker_compose() {
-	local completionCommand="__completeNoDesc"
-	local resultArray=($COMPOSE_PLUGIN_PATH $completionCommand compose)
+__completeNoDesc() {
```

Comment:
I need to check if I updated this correctly. The intent is to:

- change `/usr/local/lib/docker/cli-plugins/docker-compose __completeNoDesc <command>` to `docker __completeNoDesc <command>`
- the same should now work for all plugins, so, we should be able to just use all plugins:

        docker info --format '{{range .ClientInfo.Plugins}}{{ .Name }} {{end}}'
        buildx compose scan